clustered/server/src/main/java/org/ehcache/clustered/server/state/EhcacheStateServiceDump.java

```diff
@@ -33,10 +33,6 @@ public static void dump(EhcacheStateService clusterTierManagerState, StateDumpCo
       poolDump.addState("size", String.valueOf(poolEntry.getValue().getSize()));
     }
 
-    StateDumpCollector storesDump = clusterTierManagerStateDump.subStateDumpCollector("stores");
-    int idx = 0;
-    for (String name : new HashSet<>(clusterTierManagerState.getStores())) {
-      storesDump.addState(String.valueOf(idx++), name);
-    }
+    clusterTierManagerStateDump.addState("stores", clusterTierManagerState.getStores());
```

Comment:
Handing out internal data structures like this can be dangerous. Might want to wrap it.